react-frontend/src/components/Resources/resources.js

```diff
@@ -6,26 +6,27 @@ import Guides from './guides';
 import SimpleBanner from '../PageElements/BannerSimple/bannerSimple';
 import Standards from './standards';
 
-import { PageContainer } from '../StyleComponents/pageContent';
-
-import '../../css/resources.css';
+import {
+  PageContainerResource,
+  SimplePageBody,
+} from '../StyleComponents/pageContent';
 
 function Resources() {
   return (
-    <PageContainer>
+    <PageContainerResource>
```

Comment:
Suggest we don't need two separate things `PageContainer` and `PageContainerResource`. Can you revert back to `PageContainer` and eliminate all instances of `PageContainerResource`?

Reply:
PageContainer has been renaimed. 
As a note I now renamed the exiting page container 'PageContainerOld' with a comment stating the goal of deleting it via refactors.